### ISSUE TITLE
Missing code signature in Caprine 2.55.3 and 2.55.4

### ISSUE THREAD

**Issue by suschizu** [sindresorhus/caprine#1797]
Hello - I've received a notification that the codesignature is missing in macOS update 2.55.3 (https://github.com/sindresorhus/caprine/releases/download/v2.55.3/Caprine-2.55.3.dmg)
`EXPECTED Sindre Sorhus (YG56YK5RN5)`

```
codesign -dv --verbose=4 *pathtoapp*
result:
Caprine-2.55.4_folder/Caprine.app: code object is not signed at all
```

can you confirm this?
thanks

**Comment by core-code** [comment]
because of the security implications of the unexplained missing signature we've removed support for updating Caprine from our MacUpdater

**Comment by mquevill** [comment]
From the logs of the v2.56.1 build:
```
  • skipped macOS application code signing  reason=cannot find valid "Developer ID Application" identity or custom non-Apple code signing certificate, it could cause some undefined behaviour, e.g. macOS localized description not visible, see https://electron.build/code-signing allIdentities=  1) 90CC640E7BDBBB290BFF2A3F001D707EA5308DCE "Developer ID Application: Sindre Sorhus (YG56YK5RN5)" (CSSMERR_TP_CERT_EXPIRED)
  2) A4246F1EED66703A407491D87E0D00DE45D2E01D "3rd Party Mac Developer Installer: Sindre Sorhus (YG56YK5RN5)" (CSSMERR_TP_CERT_EXPIRED)
  3) DAABB40B7933E2539F65795B50CBE8A1CBC3C74E "3rd Party Mac Developer Application: Sindre Sorhus (YG56YK5RN5)" (CSSMERR_TP_CERT_EXPIRED)
     3 identities found
                                                Valid identities only
     0 valid identities found
```

@sindresorhus Does this developer certificate need to be updated/renewed?

**Comment by brimarq** [comment]
Just tried installing the latest v2.57.0 through homebrew - no signature. Looks like the last version with a valid code signature was v2.55.2. @sindresorhus are you just not signing these anymore?

**Comment by brimarq** [comment]
> finally resolved in 2.57.4, but the silence on an important security issue is concerning
> 
> please close

Ok, 2.57.4 is now signed; but, upon launching it in Ventura 13.4, I get a modal with the following message:

**“Caprine.app” can’t be opened because Apple cannot check it for malicious software.**
This software needs to be updated. Contact the developer for more information.

What needs to be updated?
